esrally/chart_generator.py

```diff
@@ -1364,13 +1364,23 @@ def index(environment, race_configs, title):
         }
 
 
-def load_track(cfg, name=None, track_params={}):
-    # hack to make this work with multiple tracks (Rally core is usually not meant to be used this way)
-    if name:
-        cfg.add(config.Scope.applicationOverride, "track", "track.name", name)
-    # another hack to ensure any track-params in the race config are used by Rally's track loader
-    cfg.add(config.Scope.applicationOverride, "track", "params", track_params)
-    return track.load_track(cfg)
+class RaceConfigTrack:
+    def __init__(self, cfg, name=None):
+        self.cached_track = self.load_track(cfg, name)
```

Comment:
By loading the parameterless version eagerly we would load it unneccesarily when there are only parameterized versions of track but that's probably ok.